pkg/tournament/tournament.go

```diff
@@ -896,14 +896,16 @@ func DivisionChannelName(tid, division string) string {
 }
 
 func StartRoundCountdown(ctx context.Context, ts TournamentStore, id string,
-	division string, round int, save, send bool) error {
+	division string, round int, save, send, lock bool) error {
 	t, err := ts.Get(ctx, id)
```

Comment:
there still is no guarantee that this t is the same as the one StartAllRoundCountdowns will be saving

Reply:
I think this will be OBE by the time it becomes a problem.